src/components/Avatar/Avatar.vue

```diff
@@ -100,6 +101,20 @@ import Tooltip from '../../directives/Tooltip'
 import usernameToColor from '../../functions/usernameToColor'
 import { userStatus } from '../../mixins'
 
+const browserStorage = getBuilder('nextcloud').persist().build()
```

Comment:
but now it will use `localStorage` and not `sessionStorage` – intended?

Reply:
that's fine. will increase performance for an even longer time then.